frontend/apps/erd-web/app/erd/p/[...slug]/page.tsx

```diff
@@ -23,8 +25,10 @@ export default async function Page({
 
   const input = await res.text()
 
-  // Currently supports Postgres only
-  const { value: dbStructure, errors } = await parse(input, 'postgres')
+  setPrismWasmUrl(path.resolve(process.cwd(), 'prism.wasm'))
+
+  // Currently supports schema.rb only
+  const { value: dbStructure, errors } = await parse(input, 'schemarb')
```

Comment:
Temporarily changed to support only schema.rb. 
I plan to create a PR to implement the format identification process after this.